documentation/appendix_deploying_kubernetes_openshift_cluster.adoc

```diff
@@ -0,0 +1,38 @@
+[appendix]
+== Installing Kubernetes and OpenShift cluster
+
+The easiest way to get started with Kubernetes or OpenShift is using the `Minikube`, `Minishift` or `oc cluster up`
+utilities. This section provides a basic guidance on how to use them. More details are provided on the websites of
+the tools themselves.
+
+=== Kubernetes
+
+In order to interact with a Kubernetes cluster the https://kubernetes.io/docs/tasks/tools/install-kubectl/[`kubectl`]
+utility needs to be installed.
+
+The easiest way to get a running Kubernetes cluster is using `Minikube`. `Minikube` can be downloaded and installed
```

Comment:
"... using Minikube which can be ... " instead of two separate sentences ?

Reply:
I will keep this in two sentences as discussed above.

Reply:
To me it seems to be different from the previous one. In the case of the above "Strimzi" sentences @tombentley said that the two sentences said two different things : 

> One is saying what Strimzi is, the other saying what this document is about

It seems to me that it's not the same here. Maybe in this case "The easiest way to get a running Kubernetes cluster is using `Minikube` which can be downloaded ..." works ?

Reply:
@ppatierno to be clear, the way you suggest is not at all wrong in English; it's purely a matter of style, and a lot of style boils down to preference/opinion. In this case we can expect many readers to have English as a 2nd language, and I suspect (but have no evidence) that *in general* such users would find it easier to have shorter sentences, each saying a single thing, than a longer sentence which says two things. So my preference would still be for two sentences here.